modules/serviceregistry/src/main/java/org/opencastproject/serviceregistry/impl/JobDispatcher.java

```diff
@@ -306,10 +306,14 @@ public void run() {
             continue;
           }
 
+          //Prioritization of jobs from older workflows
+          dispatchableJobs.sort(new WorkflowPriorizationComparator());
           dispatchDispatchableJobs(em, dispatchableJobs);
         } while (jobsFound);
```

Comment:
You will only have the effect of prioritizing older workflows if you have fewer than 100 jobs in the queue. The job dispatcher uses pagination and only grabs the following 100 Jobs sorted by creation date.

You are sorting only the pages and not the complete resultset.